src/main/cpp/Intake.cpp

```diff
@@ -37,6 +37,11 @@ void Intake::OnUpdate(units::second_t dt) {
         _config.IntakeMotor.transmission->SetVoltage(5_V);
       }
       break;
+      case IntakeState::kPass:
+      {
```

Comment:
you should only set voltage for a little bit, maybe until the shooter sensor senses that the object has left the robot